document/FileFormat.mdk

```diff
@@ -1029,9 +1065,11 @@ Absolute path names can reveal information that might be sensitive.
 ### `environmentVariables` property { #invocation-environmentVariables }
 
 An `invocation` object may contain a property named `environmentVariables` whose value is an object.
-The property names in this object contain the names of the environment variables in the tool's
-execution environment. The value of each property is a string containing the value of the specified
+The property names in this object shall contain the names of all the environment variables in the tool's
+execution environment. The value of each property shall be a string containing the value of the specified
 environment variable.
+If the value of the environment variable is an empty string,
```

Comment:
TIL. And on Unix, an environment variable _can_ have an empty value. I added a NOTE to that effect.

---
In reply to: [75182122](https://github.com/sarif-standard/sarif-spec/pull/256#discussion_r75182122) [](ancestors = 75182122)